pom.xml

```diff
@@ -8,7 +8,7 @@
   <!-- Project Information -->
   <groupId>tools.vitruv</groupId>
   <artifactId>parent</artifactId>
-  <version>3.1.0-SNAPSHOT</version>
+  <version>3.3.0-SNAPSHOT</version>
```

Comment:
Should be 3.2.0

Reply:
That is the new snapshot version, the release will be on the commit before this one, which is 3.2.0.